include/kodlab_mjbots_sdk/jointBase.h

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
Probably want to follow the same file naming convention I have been using or instead change everything to match